lib/manageiq/postgres_ha_admin/config_handler/rails_config_handler.rb

```diff
@@ -4,29 +4,33 @@
 
 module ManageIQ
 module PostgresHaAdmin
```

Comment:
Looks good but this lack of indenting is driving me nuts.  Did @bdunne put you up to this? 🤣 

JK, I'm fine with merging this and cleaning that up later.